unified_planning/solvers/factory.py

```diff
@@ -80,7 +84,8 @@ def _get_solver(self, solver_kind: str, name: Optional[str] = None,
     def OneshotPlanner(self, *, name: Optional[str] = None,
                        names: Optional[List[str]] = None,
                        params: Union[Dict[str, str], List[Dict[str, str]]] = None,
-                       problem_kind: ProblemKind = ProblemKind()) -> Optional['unified_planning.solvers.solver.Solver']:
+                       problem_kind: ProblemKind = ProblemKind(),
+                       optimality_guarantee: Optional[int] = None) -> Optional['up.solvers.solver.Solver']:
```

Comment:
I think it would be useful to also support the strings `"OPTIMAL"` and `"SATISFICING"` in addition to the INT values